home.html

```diff
@@ -0,0 +1,201 @@
+        </div>
+      </main>
+      <section class="program">
+        <div class="title-program">
+          <h5>Program</h5>
+        </div>
+        <hr />
+        <ul class="mobile-ul">
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/presentation.svg"
+              alt="Icon presentation"
+            />
+            <h6>Lecture</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/exhibition.svg"
+              alt="Icon presentation"
+            />
+            <h6>Exhibition</h6>
+            <p class="paragraph-program">
+              Listen to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/discussion.svg"
+              alt="Icon presentation"
+            />
+            <h6>Forum</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/workshop.svg"
+              alt="Icon presentation"
+            />
+            <h6>Workshop</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/dharma-wheel.svg"
+              alt="Icon presentation"
+            />
+            <h6>Dharma</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+        </ul>
+        <div class="button-program">
+          <button type="button" class="btn-btn">Join Estudio Polanco</button>
+        </div>
+        <div class="link-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section class="models">
+        <div class="title-models">
+          <h4>Featured Models</h4>
+        </div>
+        <hr />
+        <ul class="models-list"></ul>
+      </section>
+      <div class="button-more">
+        <button type="button">MORE &#5167;</button>
+      </div>
+      <div class="desktop-logo">
+        <div class="title-about-four">
+          <h5>Partners</h5>
+          <hr />
+        </div>
+        <div class="logos">
```

Comment:
- [ ] This section is not aligned correctly and items have too much margin between them. You should place them correctly and reduce the margin between them. 

The Project :arrow_down: 
![Screenshot from 2021-07-03 00-58-01](https://user-images.githubusercontent.com/1642014/124335324-95d6cf80-dba2-11eb-88fd-595ace67466b.png)

The Design Template :arrow_down: 
![Screenshot 2021-07-03 at 02-01-42 CC Global Summit 2015](https://user-images.githubusercontent.com/1642014/124335347-abe49000-dba2-11eb-9c21-d0e23348959a.png)